Tensile/Tests/unit/test_mergeLogic.py

```diff
@@ -52,8 +55,10 @@
 -
   - SolutionIndex: 39
     SolutionNameMin: InUseForSize256or1024xxx
+    InUseForSize256or1024xxx: 0
```

Comment:
I think these added lines are meant for the merge script to determine unique solutions, now that index and name are ignored during comparison. So perhaps something like DummyParam will make the intent clearer. Other than that, looks good to me!

Reply:
This is exactly correct; the latest commit hopefully makes this intent more clear.